xmr-btc/tests/e2e.rs

```diff
@@ -152,12 +152,12 @@ mod tests {
         node::{run_alice_until, run_bob_until},
     };
 
-    use futures::future;
     use monero_harness::Monero;
     use rand::rngs::OsRng;
-    use std::convert::TryInto;
     use testcontainers::clients::Cli;
 
```

Comment:
Please do not leave whitespace between use statements unless it implies some meaning (such as separating current repository libraries from other libraries).